Products/products.css

```diff
@@ -0,0 +1,74 @@
+.product-handler {
+    display: flex;
+    gap: 1rem;
+    background-color: var(--background-color);
+}
+
+.products-sidebar {
+    width: max-content;
+    padding: 3rem;
+    background-color: var(--warning-color);
+    border-radius: 5px;
+    width: 35rem;
+}
+
+.sidebar-list {
+    display: flex;
+    flex-direction: column;
+    font-size: 1.2rem;
+    font-weight: 500;
+}
+
+.display {
+    display: flex;
+}
+
+.sidebar-heading {
+    font-size: 1.7rem;
+    font-weight: 600;
+    margin: 1rem 0rem;
```

Comment:
you can extract margins and padding in the utility classes.

Reply:
I will keep in mind your suggestions for the next time while coding